src/cosmicqc/frame.py

```diff
@@ -591,24 +604,87 @@ def find_image_columns(self: CytoDataFrame_type) -> bool:
         ]
 
     @staticmethod
+    def draw_outline_on_image(actual_image_path: str, mask_image_path: str) -> Image:
+        """
+        Draws outlines on a TIFF image based on a mask image and returns
+        the combined result.
+
+        This method takes the path to a TIFF image and a mask image, creates
+        an outline from the mask, and overlays it on the TIFF image. The resulting
+        image, which combines the TIFF image with the mask outline, is returned.
+
+        Args:
+            actual_image_path (str): Path to the TIFF image file.
+            mask_image_path (str): Path to the mask image file.
+
+        Returns:
+            PIL.Image.Image: A PIL Image object that is the result of
+            combining the TIFF image with the mask outline.
+
+        Raises:
+            FileNotFoundError: If the specified image or mask file does not exist.
+            ValueError: If the images are not in compatible formats or sizes.
+        """
+        # Load the TIFF image and convert it to grayscale array
+        tiff_image_array = skimage.io.imread(actual_image_path)
+        # Convert to PIL Image and then to 'RGBA'
```

Comment:
What is RGBA?

Reply:
Thanks, good question - [RGBA is defined](https://pillow.readthedocs.io/en/stable/handbook/concepts.html#modes) as: `"(4x8-bit pixels, true color with transparency mask)"`. This ended up being the simplest way this seemed to work, but open to other methods too if you think another mode might work better!